app/templates/components/payments/funds-recipient/details-form.hbs

```diff
@@ -45,7 +45,12 @@
   <div class="input-section__label">Birthdate</div>
   <div class="input-section__content">
     <div class="input-group">
-      {{select/birth-date day=stripeConnectAccount.legalEntityDobDay month=stripeConnectAccount.legalEntityDobMonth year=stripeConnectAccount.legalEntityDobYear}}
+      {{select/birth-date
+        day=stripeConnectAccount.legalEntityDobDay
+        month=stripeConnectAccount.legalEntityDobMonth
+        year=stripeConnectAccount.legalEntityDobYear
+        onChange=(action onBirthDateChanged)
```

Comment:
just using a `mut` helper like with country and state below would be too complex here, so had to resort to calling a component function.